lib/matplotlib/tests/test_axes.py

```diff
@@ -982,6 +982,27 @@ def test_canonical():
     ax.plot([1, 2, 3])
 
 
+@image_comparison(baseline_images=['arc_angles'], remove_text=True,
+                  style='default')
+def test_arc_angles():
+    from matplotlib import patches
+    # Ellipse parameters
+    w = 2
+    h = 1
+    centre = (0.2, 0.5)
+
+    fig, axs = plt.subplots(3, 3)
+    for i, ax in enumerate(np.ravel(axs)):
+        theta2 = i * 360 / 9
+        theta1 = theta2 - 45
+        ax.add_patch(patches.Arc(centre, w, h, theta1=theta1, theta2=theta2))
+        # Straight line should intersect end of arc
```

Comment:
Could you also add the start of the arc as well? Start -> origin -> end should work.